examples/morpheusvm/config/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	// Tracing
 	TraceEnabled    bool    `json:"traceEnabled"`
 	TraceSampleRate float64 `json:"traceSampleRate"`
+	DSN             string  `json:"dsn"`
```

Comment:
To confirm, this is provided by Uptrace on startup?

Reply:
Yes it is provided by the running uptrace.
It contains a secret to communicate with uptrace, is it safe to use it in this config?

Reply:
We can set the default value (like I did in my patch).